src/bundle/Resources/public/js/scripts/admin.input.text.js

```diff
@@ -1,5 +1,4 @@
-(function (global, doc, $, ibexa) {
-    const textInputClearBtns = doc.querySelectorAll('.ibexa-input-text-wrapper__action-btn--clear');
+(function(global, doc, $, ibexa) {
```

Comment:
Do we need `$`?

Reply:
Eh, come on, I just deleted extra space after function ` `. xD

Reply:
Deleted 😄